ml/Config.cc

```diff
@@ -31,7 +31,7 @@ void Config::readMLConfig(void) {
     unsigned MaxTrainSamples = config_get_number(ConfigSectionML, "maximum num samples to train", 4 * 3600);
     unsigned MinTrainSamples = config_get_number(ConfigSectionML, "minimum num samples to train", 1 * 900);
     unsigned TrainEvery = config_get_number(ConfigSectionML, "train every", 1 * 3600);
-    unsigned NumModelsToUse = config_get_number(ConfigSectionML, "number of models per dimension", 1 * 24);
+    unsigned NumModelsToUse = config_get_number(ConfigSectionML, "number of models per dimension", 1);
```

Comment:
makes sense to leave this like this (defaulting to just 1 model) and i will make a follow on PR to change the defaults on a few weeks after i dogfood my new proposed defaults a bit more first.